compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.td

```diff
@@ -71,7 +72,8 @@ def DispatchLoweringPassPipelineEnum
             CPU_TripleTilingExpert, CPU_DataTiling, LLVMGPU_SimpleDistribute,
             LLVMGPU_Vectorize, LLVMGPU_MatmulSimt, LLVMGPU_MatmulTensorCore,
```

Comment:
Since we added `MmaSync` as a suffix to `LLVMGPU_MatmulTensorCore`, would it make sense to add `Wmma` `LLVMGPU_MatmulTensorCore` , i.e., rename `LLVMGPU_MatmulTensorCore` to `LLVMGPU_MatmulTensorCoreWmma`. `LLVMGPU_MatmulTensorCore` makes me feel that it includes both `wmma` Tensor Cores and `mma.sync` Tensor Cores.

Reply:
It would but that would break the existing tuning parameters if we changed the string name, so I would prefer doing that after we transition.

Reply:
ok. We can keep it as `LLVMGPU_MatmulTensorCore` and we can try and apply another renaming patch if it makes sense.